bitnami/minio/templates/provisioning-configmap.yaml

```diff
@@ -1,11 +1,11 @@
 {{- if .Values.provisioning.enabled }}
 {{- $fullname := printf "%s-provisioning" (include "common.names.fullname" .) }}
-{{- $releaseNamespace := .Release.Namespace | quote }}
+{{- $releaseNamespace := .Release.Namespace }}
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ $fullname }}
-  namespace: {{ $releaseNamespace }}
+  namespace: {{ $releaseNamespace | quote }}
```

Comment:
Actually, the only use of `$releaseNamespace` is here so we can remove the created variable in line 3 and use the following:
```
{{- $fullname := printf "%s-provisioning" (include "common.names.fullname" .) }}
apiVersion: v1
kind: ConfigMap
metadata:
  name: {{ $fullname }}
  namespace: {{ .Release.Namespace | quote }}
```